ocaml/idl/datamodel_host.ml

```diff
@@ -1011,9 +1011,19 @@ let host_query_ha = call ~flags:[`Session]
       ~allowed_roles:_R_POOL_ADMIN
       ()
 
+  let reset_server_certificate = call
+      ~flags:[`Session]
+      ~lifecycle:[Published, rel_next, ""]
+      ~name:"reset_server_certificate"
+      ~doc:"Delete the current TLS server certificate and replace by a new, self-signed one. This should only be used with extreme care."
+      ~versioned_params: []
```

Comment:
The call is missing a `host` parameter with type `Ref _host`. Before since the call was only local this parameter was implicit.